lib/SIL/Parser/ParseSIL.cpp

```diff
@@ -699,7 +699,7 @@ static bool parseDeclSILOptional(
       *serializedKind = IsSerialized;
     else if (serializedKind && SP.P.Tok.getText() == "serialized_for_package")
       *serializedKind = IsSerializedForPackage;
-    else if (isDynamic && SP.P.Tok.getText() == "dynamically_replacable")
```

Comment:
@AnthonyLatsis should revert right?

Reply:
https://github.com/swiftlang/swift/pull/75020#discussion_r1711289572.

Reply:
I'm leaving it as is.